JSDemos/Demos/DataGrid/RecordPaging/description.md

```diff
@@ -1 +1,12 @@
-The **DataGrid** widget can automatically split its content across multiple pages and allow end users to&nbsp;navigate records using its built-in pager. This demo illustrates how you can change pager settings by&nbsp;using both the **paging** and **pager** configuration options.
\ No newline at end of file
+The **DataGrid** splits records into multiple pages. This technique helps optimize the control's performance: the client only needs to load and render one page at a time. Users can use a scroll bar or a pager to navigate between pages. This demo shows how to enable and customize the pager.
```

Comment:
_This technique helps optimize the control's performance: the client only needs to load and render one page at a time. Users can use a scroll bar or a pager to navigate between pages._

Is this a "technique", or do you mean "feature"?

Reply:
I would say it is more a technique than a feature.

Reply:
```suggestion
The **DataGrid** splits records into multiple pages. This technique helps optimize control performance: the client only needs to load and render one page at a time. Users can use a scroll bar or a pager to navigate between pages. This demo shows how to enable and customize the pager.
```